test/system/helpers.bash

```diff
@@ -67,7 +67,7 @@ function basic_teardown() {
     run_podman '?' pod rm --all --force
     run_podman '?'     rm --all --force
 
-    /bin/rm -rf $PODMAN_TMPDIR
```

Comment:
The intention here was to override a possible `rm` alias. We once had CI break because of that (on this project), and pretty much everyone has hit that problem at least once in their career. I might feel more comfortable with `command rm`; but it's probably not worth incurring the wrath of CI to resubmit.

I know you must get this a lot, but it's really offensive for a distro to break fifty years of UNIX `/bin` precedent. I'm not blaming you, I know you're trying to work with the framework you have. I just need to vent.

Reply:
Alright I changed it to `$(command -v rm) …`, I think this would also work if one has set an alias for `rm`. :innocent:

Reply:
Wait, why not just `command rm`? `$(command -v rm)` seems overly rococo?

Reply:
Makes sense, fixed that too :see_no_evil:

Reply:
You could also just to \rm

Reply:
You could also just to \rm